tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -2259,3 +2277,313 @@ describe("clipboard: pasting outside of sheet", () => {
+
+    setCellContent(modelA, "B2", "b2");
+    setStyle(modelA, "B2", cellStyle);
+
+    expect(getCell(modelA, "B2")).toMatchObject({
+      content: "b2",
+      style: cellStyle,
+    });
+
+    selectCell(modelA, "B2");
+    doAction(["edit", "copy"], envA);
+
+    /**
+     * Copy the clipboard from envA to envB because
+     * in this context we need to simulate that
+     * the clipboard is shared between the two environments
+     * given that in a real world scenario we are using one
+     * clipboard which is the machine clipboard.
+     */
+    envB.clipboard = envA.clipboard;
+
+    selectCell(modelB, "D2");
+    await doAction(["edit", "paste"], envB);
+
+    expect(getCell(modelA, "B2")).toMatchObject({
+      content: "b2",
+    });
+    expect(getCell(modelB, "D2")).toMatchObject({
+      content: "b2",
+    });
+    expect(getStyle(modelA, "B2")).toMatchObject(cellStyle);
+    expect(getStyle(modelB, "D2")).toMatchObject(cellStyle);
+  });
+
+  test("it should copy/paste a cell with the cell content and format copied last from an external spreadsheet", () => {
+    const modelA = new Model();
+    const modelB = new Model();
+    const cellStyle = { bold: true, fillColor: "#00FF00", fontSize: 20 };
+
+    setCellContent(modelA, "A1", "a1");
+    setStyle(modelA, "A1", cellStyle);
+    setCellContent(modelB, "C1", "c1");
+    setStyle(modelB, "C1", cellStyle);
+
+    expect(getCell(modelA, "A1")).toMatchObject({
+      content: "a1",
+      style: cellStyle,
+    });
+
+    expect(getCell(modelB, "C1")).toMatchObject({
+      content: "c1",
+      style: cellStyle,
+    });
+
+    copy(modelB, "C1");
+    copy(modelA, "A1");
+    const clipboardContent = modelA.getters.getClipboardContent();
+
+    expect(clipboardContent["text/plain"]).toBe("a1");
+
+    pasteFromOSClipboard(modelB, "B1", {
+      [ClipboardMIMEType.PlainText]: clipboardContent["text/plain"]
+        ? clipboardContent["text/plain"]
+        : "",
+      [ClipboardMIMEType.OSpreadsheet]: clipboardContent["web application/o-spreadsheet"],
+    });
+
+    expect(getCell(modelA, "A1")).toMatchObject({
+      content: "a1",
+    });
+    expect(getCell(modelB, "B1")).toMatchObject({
+      content: "a1",
+    });
+    expect(getStyle(modelA, "A1")).toMatchObject(cellStyle);
+    expect(getStyle(modelB, "B1")).toMatchObject(cellStyle);
+  });
+
+  test("it should copy/paste a formula cell with dependencies", () => {
+    const modelA = new Model({ sheets: [{ id: "sheetA" }] });
+    const modelB = new Model({ sheets: [{ id: "sheetB" }] });
+
+    setCellContent(modelA, "A1", "1");
+    setCellContent(modelA, "A2", "2");
+    setCellContent(modelA, "A3", "=SUM(1,2)");
+    setCellContent(modelA, "B1", "4");
+    setCellContent(modelA, "B2", "5");
+    setCellContent(modelA, "B3", "6");
+    setCellContent(modelA, "C1", "=A1*B1");
+    setCellContent(modelA, "C2", "=A2*B2");
+    setCellContent(modelA, "C3", "=A3*B3");
```

Comment:
if the test title says `a formula cell with dependencies`, then there's no reason to add cells without dependencies.
Also testing 3 times the same (almost) formula is useless 
```suggestion
    setCellContent(modelA, "C3", "=A3*B3");
```